app/utils/behavior/compute.js

```diff
@@ -0,0 +1,73 @@
+import * as ss from 'simple-statistics';
+
+export const aggregateData = (rawData, dv, files) => {
```

Comment:
What's `dv`? Is it short for data value? Might want to use a more descriptive variable name

Reply:
It was dependent variable:) I've changed the name to "dependentVariable"